plugins/usbdmx/DMXCProjectsNodleU1Factory.cpp

```diff
@@ -30,14 +30,25 @@ namespace ola {
 namespace plugin {
 namespace usbdmx {
 
-const uint16_t DMXCProjectsNodleU1Factory::VENDOR_ID = 0x16d0;
-const uint16_t DMXCProjectsNodleU1Factory::PRODUCT_ID = 0x0830;
+const uint16_t DMXCProjectsNodleU1Factory::VENDOR_ID_NODLE_U1 = 0x16d0;
```

Comment:
I still wonder if just vendor name in the vendor IDs, and then vendor and product name in the product IDs would make sense.

Reply:
I initially didn't understand what you mean. But I guess my comment got lost as I did push some changes. The naming makes sense. But vendor+product break the max line length in linting. Maybe just product name for the product ID?

Reply:
Which bit of it breaks the line length and where? We may be able to do some line wrap stuff or something, or come up with a slightly more concise name. Can you perhaps comment out the existing names and stick some new ones in so we can see how it looks.

Reply:
Done in 30577e3cc69a8acf161f5f1e6dd6a8e617d9b704